Assignment 2/180501131/attendanceStudents.py

```diff
@@ -0,0 +1,5 @@
+attdlist = ['Leena-P','Steve-A','Dinesh-P','Pragya-P']
+print("The present students were:")
+for e in attdlist:
+    if e[-1]=='P':
+        print(e)
```

Comment:
Make the code more dynamic. It should be able to accept any set of inputs. Feel free to include extra inputs that will help you get this done like the number of students. Also, the output must only be the student's name not their attendance